documents/refund.md

```diff
@@ -0,0 +1,307 @@
+## Refunds
+
+```rb
+require "razorpay"
+
+Razorpay.setup('key_id', 'key_secret')
+```
+
+### Create a normal refund
+
+```rb
+Razorpay::QrCode.create(paymentId,{
+  "amount": "100",
+  "speed": "normal",
+  "notes": {
+    "notes_key_1": "Beam me up Scotty.",
+    "notes_key_2": "Engage"
+  },
+  "receipt": "Receipt No. 31"
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+|  paymentId*   | string      | The id of the payment                       |
+|  amount       | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |                       |
+|  speed        | string      | Here, it must be normal                |
+|  notes        | array       | A key-value pair                |
+|  receipt      | string      | A unique identifier provided by you for your internal reference. |
+
+**Response:**
+```json
+{
+  "id": "rfnd_FP8QHiV938haTz",
+  "entity": "refund",
+  "amount": 500100,
+  "receipt": "Receipt No. 31",
+  "currency": "INR",
+  "payment_id": "pay_FCXKPFtYfPXJPy",
+  "notes": [],
+  "acquirer_data": {
+    "arn": null
+  },
+  "created_at": 1597078866,
+  "batch_id": null,
+  "status": "processed",
+  "speed_processed": "normal"
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Create an instant refund
+
+```rb
+Razorpay::QrCode.create(paymentId,{
+  "amount": "100",
+  "speed": "optimum",
+  "receipt": "Receipt No. 31"
+})
+```
```

Comment:
Not part of `lib/razorpay/refund.rb`. And why `Razorpay::QrCode` to raise a instant refund request?